.travis.yml

```diff
@@ -2,12 +2,19 @@ language: scala
 jdk: oraclejdk7
 sudo: false
 before_install: umask 0022
+cache:
+  directories:
+   - $HOME/.scalding.dep.cache
 script:
   - "echo no op"
+addons:
+  apt:
+    packages:
+    - md5deep
 matrix:
   include:
 #BASE TESTS
```

Comment:
I'm nervous we need a test to be sure we are running all the tests. It would be really easy to leave a target out. Any ideas on how we can be sure we have everything other than just manually checking that all the targets appear somewhere?